### DISCUSSION TITLE
Partial stats with terminated job on windows

### DISCUSSION THREAD

**Discussion by gshum** [axboe/fio#1819] [General]
Hi,
I have a script that is running an FIO workload. When I terminate this workload with ctl+c (windows) it will print a summary to the bottom of the output file and I can be 100% sure it was running correctly.

However, I have to use taskkill or stop-process (powershell) to kill the process as part of my script. Then I get no summary in the output file.

Is there any option or way that I can run some fio job for some indeterminate time, kill the job, and still get some sort of IOPs summary on a windows based computer?

Thanks a lot for any advice.

**Comment by brandon-paupore-sndk** [comment]
Hmm, for reference Windows has a different signal SIGBREAK for taskkill and similar functions that terminate another process.
https://github.com/axboe/fio/pull/1450

FIO handles it mostly the same as SIGINT from Ctrl+C, with the main difference being a delay to prevent Windows from terminating the entire FIO process after the signal is handled. Currently this is done by checking that each thread has exited and is waiting on the core to reap it, but maybe that could be extended to allow the summary to complete.

**Reply by gshum** [reply]
Thats useful information. I found a solution by adding --status-interval=60.

This writes the test status to the output file every 60 seconds so that even if I terminate the process I can be sure it was running correctly after the act.

**Comment by vincentkfu** [comment]
The PR linked by Brandon has instructions for obtaining output by killing the command prompt window within which fio is running. Does that work for you?
